build.sbt

```diff
@@ -1,27 +1,33 @@
 val orgName = "Marshall Wace"
 val year = 2023
 
-ThisBuild / scalaVersion        := "3.3.1"
-ThisBuild / version             := "0.1.0-SNAPSHOT"
-ThisBuild / organization        := "com.mwam.kafkakewl"
-ThisBuild / organizationName    := orgName
-ThisBuild / startYear           := Some(year)
-
-ThisBuild / scalacOptions       ++= Seq(
-  "-Xmax-inlines", "64",
+ThisBuild / scalaVersion := "3.3.0"
+ThisBuild / version := "0.1.0-SNAPSHOT"
+ThisBuild / organization := "com.mwam.kafkakewl"
+ThisBuild / organizationName := orgName
+ThisBuild / startYear := Some(year)
+
+ThisBuild / scalacOptions ++= Seq(
+  "-Xmax-inlines",
+  "64",
   "-Yretain-trees", // so that zio-json supports default values
-  "-Wunused:imports", "-Wunused:params", "-deprecation", "-feature"
+  "-Wunused:imports",
```

Comment:
these changes exist in the other PR too: https://github.com/MarshallWace/kafkakewl/pull/47/files#diff-5634c415cd8c8504fdb973a3ed092300b43c4b8fc1e184f7249eb29a55511f91R14
In cases like these I make the other branch the target instead of master so it doesn't show as a diff twice in PRs.